common/djangoapps/enrollment/tests/test_views.py

```diff
@@ -32,6 +32,15 @@
 from embargo.test_utils import restrict_course
 
 
+class UrlMixin(object):
+    """
+    Mixins for URL processing
+    """
+    def get_absolute_url(self, path):
+        """ Generate an absolute URL for a resource on the test server. """
+        return u'http://testserver/{}'.format(path.lstrip('/'))
+
+
```

Comment:
I agree with using a utility function instead of a mixin. We are not overriding the hostname (testserver) for LMS tests. I'm not sure it can be overridden, so we should be safe hard-coding that value. @Nickersoft let me know if you need assistance  with this change. I would move this code to `openedx/core/lib/django_test_client_utils.py`.